app/bundles/FormBundle/Form/Type/FieldType.php

```diff
@@ -317,6 +319,30 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             );
         }
 
+        if ($addFieldWidth) {
+            $builder->add(
+                'fieldWidth',
+                ChoiceType::class,
+                [
+                    'label'      => 'mautic.form.field.form.field_width',
+                    'label_attr' => ['class' => 'control-label'],
+                    'attr'       => [
+                        'class'   => 'form-control',
+                        'tooltip' => 'mautic.form.field.help.field_width',
+                    ],
+                    'choices' => [
+                        'mautic.form.field.form.field_width.full'           => 'full',
+                        'mautic.form.field.form.field_width.three_quarters' => 'three-quarters',
```

Comment:
Can we use direct percentages instead of words and use those numbers directly in the `style` attr instead of generating a CSS class that then have a width?
```suggestion
                        'mautic.form.field.form.field_width.full'           => '100',
                        'mautic.form.field.form.field_width.three_quarters' => '75',
```
...